.github/workflows/web-actions.yml

```diff
@@ -134,4 +134,5 @@ jobs:
           kubectl set env deployment/cc-web-deploy CHAT_PROVIDER=openai
           kubectl set env deployment/cc-web-deploy OPENAI_API_KEY=${{secrets.OPENAI_API_KEY}}
           kubectl set env deployment/cc-web-deploy HUGGINGFACE_API_KEY=${{secrets.HUGGINGFACE_API_KEY}}
+          kubectl set env deployment/cc-web-deploy ADMIN_EMAILS="greta@openearth.org,ayinawu@openearth.org,amanda@openarth.org"
```

Comment:
These should be moved to k8s/cc-web-deploy.yml. All non-secret env vars are currently configured there. We'll move the CHAT_PROVIDER entry there as well.

Reply:
Also you're not configuring ADMIN_NAMES here, which is mentioned in env.example.